cmd/bashbrew/cmd-cat.go

```diff
@@ -56,6 +56,13 @@ func cmdCat(c *cli.Context) error {
 		"arch": func() string {
 			return arch
 		},
+		"gitCache": func() (string, error) {
+			err := ensureGitInit()
+			if err != nil {
+				return "", err
+			}
+			return gitCache(), nil
+		},
```

Comment:
This one added function covers at least 95% of the cases where I set `BASHBREW_CACHE` in scripts :see_no_evil:

It also removes the need for `bashbrew from hello-world` that I do in a lot of those place (whose purpose is just to make sure `ensureGitInit` has been invoked :sob:).